spec/puppetserver/ca/action/prune_spec.rb

```diff
@@ -95,13 +95,21 @@
   end
 
   describe 'update' do
-    it 'bumps CRL version up by 1' do
+    it 'bumps CRL number up by 1' do
       ca_key = OpenSSL::PKey::RSA.new(512)
       ca_cert = create_cert(ca_key, "Bazzup")
       ca_crl = create_crl(ca_cert, ca_key)
 
       subject.update_pruned_CRL([ca_crl], ca_key)
-      expect(ca_crl.version).to eq(2)
+
+      # crlNumber is based on the amount of certs in revoked list
+      # in our case, that number is 0. So crlNumber is 0.
+      ca_crl.extensions.each do |ext|
+        if ext.oid == "crlNumber"
+          ext.value=((ext.value.to_i + 1).to_s)
```

Comment:
This is the same thing you're doing in `update_pruned_CRL`, so not sure how this is testing that the method does the right thing.. In fact, if we're starting with 0 as the CRL number, running `update_pruned_CRL`, and then running this, I would expect the result to be 2???